clients/webclient/src/components/Users/CustomerProfile.vue

```diff
@@ -178,6 +171,11 @@ export default class CustomerProfile extends Vue {
         this.editUserprofile = false;
         this.showUserprofile = true;
     }
+
+    save() {
+        this.showUserprofile = true;
```

Comment:
Disse to boolean er alltid motsatt av hverandre og ser ut til å brukes kun til å skifte mellom to view, så da er det bedre å bruke én boolean sammen med v-if og v-else i template.